cibyl/exceptions/config.py

```diff
@@ -30,3 +30,11 @@ def __init__(self, message="""
             system_type: jenkins"""):
         self.message = message
         super().__init__(self.message)
+
+
+class ConfigurationNotFound(CibylException):
+    """Configuration file not found exception"""
+    def __init__(self, message="Could not find configuration file:"
+                 "('~/.config/cibyl.yaml', '/etc/cibyl/cibyl.yaml')"):
```

Comment:
If you prefer, the searched paths are listed at the 'Config.DEFAULT_FILE_PATHS' constant.

Reply:
It's a good idea, but importing Config here would create a circular import